Tools/RemoteConsole/ly_remote_console/ly_remote_console/remote_console_commands.py

```diff
@@ -295,14 +296,7 @@ def _handle_message(self, message):
                     self.handlers[key].set()
                     continue
 
-        # The very first connection using the socket will return all of the auto complete items, turned off so no one
-        # wouldn't need to see them
-        elif message_type == CONSOLE_MESSAGE_MAP['AUTOCOMPLETELIST']:
```

Comment:
I assume this no longer happens?

Reply:
I left the functionality of the Auto Complete List in there as I'm not sure of its use, but I'm no longer using it as a verification that the RC is connected.